build/index.html

```diff
@@ -544,63 +544,291 @@ <h2>Govcy Inputs</h2>
+                        <input class="govcy-radio-input" type="radio" name="radio">
+                        <span class="govcy-radio" tabindex="1"></span>
+                    </label>
+                    <label class="govcy-radio-btn govcy-d-sm-inline-block">Option 3
+                        <input class="govcy-radio-input" type="radio" name="radio">
+                        <span class="govcy-radio" tabindex="2"></span>
+                    </label>
                 </div>
+            </fieldset>
+        </form>
+    </div>
+    <br><br>
+
+    <h2>Date of birth</h2>
+    <div class="govcy-container">
+        <form action="" class="govcy-form">
+            <fieldset class="govcy-fieldset">
                 <div class="govcy-form-control">
-                    <label class="govcy-label" for="">Postal code</label>
-                    <input id="" type="text" class="govcy-text-input" placeholder="Postal code" />
+                    <label class="govcy-label govcy-mb-1" for="">Date of birth</label>
+                    <span class="govcy-hint govcy-mb-3">For emample&#44; 12 11 2007</span>
+                    <div class="govcy-d-flex">
+                        <div class="govcy-d-block">
+                            <label class="govcy-label govcy-mb-1 govcy-fw-normal govcy-mb-2" for="">Day</label>
+                            <input type="text" class="govcy-text-input govcy-text-input-char_3 govcy-p-2" placeholder="" maxlength="2" />
+                        </div>
+                        <div class="govcy-d-block govcy-ml-2">
+                            <label class="govcy-label govcy-mb-1 govcy-fw-normal govcy-mb-2" for="">Month</label>
+                            <input type="text" class="govcy-text-input govcy-text-input-char_3 govcy-p-2" placeholder="" maxlength="2" />
+                        </div>
+                        <div class="govcy-d-block govcy-ml-2">
+                            <label class="govcy-label govcy-mb-1 govcy-fw-normal govcy-mb-2" for="">Year</label>
+                            <input type="text" class="govcy-text-input govcy-text-input-char_6 govcy-p-3" placeholder="" maxlength="4" />
+                        </div>
+                    </div>
+                </div>
+            </fieldset>
+        </form>
+
+        <form action="" class="govcy-form">
+            <fieldset class="govcy-fieldset">
+                <div class="govcy-form-control govcy-form-control-error">
+                    <label class="govcy-label govcy-mb-1" for="">Date of birth</label>
+                    <span class="govcy-input-error-msg govcy-mb-3">There is an error</span>
+                    <div class="govcy-d-flex">
+                        <div class="govcy-d-block">
+                            <label class="govcy-label govcy-mb-1 govcy-fw-normal govcy-mb-2" for="">Day</label>
+                            <input type="text" class="govcy-text-input govcy-text-input-char_3 govcy-p-2" placeholder="" maxlength="2" />
```

Comment:
When there is an error, we should also add the `govcy-text-input-error` class on the day, month and year text inputs, so that their border becomes red.

Reply:
govcy-text-input-error class has been added on day month and year text inputs and its ready for testing.

Reply:
👍